.github/workflows/clang-tidy.yml

```diff
@@ -80,17 +81,16 @@ jobs:
         TILES: 1
         SOUND: 1
         LOCALIZE: 1
+    if: ${{ needs.skip-duplicates.outputs.should_skip != 'true' && github.event_name != 'pull_request' || github.event.pull_request.draft == false }}
```

Comment:
Drive-by change: Skipe the whole job when the PR is in draft and show it as "skipped" instead of skipping the individual steps and showing it as ✔️ passed (which is misleading)